microservices-connector/internal/controller/gmconnector_controller.go

```diff
@@ -366,7 +349,10 @@ func (r *GMConnectorReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	// get the router config
 	// r.Log.Info("Reconciling connector graph", "apiVersion", graph.APIVersion, "graph", graph.Name)
 	fmt.Println("Reconciling connector graph", "apiVersion", graph.APIVersion, "graph", graph.Name)
-
+	err := reconcileResource("Configmap", req.NamespacedName.Namespace, "", nil, nil)
```

Comment:
think we need add logic to select 'Configmap' or 'ConfigmapGaudi' based on current tgi svc setting

Reply:
As we discussed, let's keep this and then change them based on the discussion with Oneclick.